en-US/Punctuation.xml

```diff
@@ -647,6 +767,18 @@ PLAY [Gather and display facts for managed nodes] ****************************</
 		  <para>
             The path contains the library qualifier in braces.
           </para>
+          </example>
+		<example><title>Referring to Special Characters</title>
+
+          <para>
+			To add a node, click the plus character (+).
```

Comment:
This example may not be the clearest. In our content, this would commonly use "UI" markup?

(Curiosity, non-blocking at all.)

Reply:
Maybe:

> When a regular user starts a shell, the prompt includes an ending dollar character ($).

(from RH124).

Reply:
Good example, now implemented.